lab_1_keywords_tfidf/main.py

```diff
@@ -33,6 +42,14 @@ def remove_stop_words(tokens: list[str], stop_words: list[str]) -> Optional[list
 
     In case of corrupt input arguments, None is returned
     """
+    if not isinstance(tokens, list) or not isinstance(stop_words, list):
+        return None
+    for i in tokens:
+        for j in stop_words:
```

Comment:
I believe your implementation works, but let's try to make it more pythonic - for example, use the `i in stop_words` construct

Reply:
done